Sources/eppo/EppoClient.swift

```diff
@@ -105,7 +151,22 @@ public class EppoClient {
     // This function can be called from multiple threads; synchronization is provided to safely update
     // the configuration cache but each invocation will execute a new network request with billing impact.
     public func load() async throws {
-        try await self.configurationStore.fetchAndStoreConfigurations()
+        let configuration = try await self.configurationRequester.fetchConfigurations()
+        self.configurationStore.setConfigurations(config: configuration)
+    }
+    
+    public func takeOnline(        
+        sdkKey: String,
+        host: String = default_host
+    ) {
+        let httpClient = NetworkEppoHttpClient(
+            baseURL: host,
+            sdkKey: sdkKey,
+            sdkName: sdkName,
+            sdkVersion: sdkVersion
+        )
+        self.isConfigObfuscated = true
```

Comment:
🐛 this changes config obfuscation _before_ new configuration have been fetched, which will make the client re-interpret previously supplied configuration as obfuscated even if it is not (until the client successfully fetches new configuration)